app/__init__.py

```diff
@@ -45,23 +44,21 @@
     allow_origin=app.config['ALLOW_ORIGIN'],
 )
 
-load_config()
-
 if "pytest" in sys.modules:
     from simplekv.memory import DictStore
     store = DictStore()
 else:
     store = RedisStore(redis.StrictRedis(host=app.config['REDIS_HOST']))
 
 prefixed_store = PrefixDecorator('sessions_', store)
-KVSessionExtension(prefixed_store, app)
+KVSessionExtension(prefixed_store, current_app)
```

Comment:
```suggestion
KVSessionExtension(prefixed_store, app)
```
we are not in app context here

Reply:
Turns out the solution is to not pass in the `app` or `current_app` explicitly and just let KVSession access the app_context when during the handling of a specific request.